apps/src/code-studio/stageLockRedux.js

```diff
@@ -110,6 +119,12 @@ export default function reducer(state = initialState, action) {
 }
 
 // Action creators
+
+/**
+ * Authorizes the user to be able to see lockable stages
+ */
+export const authorizeLockable = () => ({ type: AUTHORIZE_LOCKABLE });
```

Comment:
Though I like the look of how I did it better, their reasoning for why they prefer their approach has some merit. This PR doesn't seem like the right place to convert this file, but I may do so in a future commit/may start favoring their approach for new files.